### ISSUE TITLE
bug(creatable): cannot render AutoCompleteCreatable

### ISSUE THREAD

**Issue by eric-hc** [anubra266/choc-autocomplete#58]
Following these docs:

```md
First add the `creatable` prop to the AutoComplete component. Then add the `AutoCompleteCreatable` component to the bottom of the list. Refer to the references for more info on this component.
```

I was not able to render the `<AutoCompleteCreatable>` component within the `AutoCompleteList`

**Comment by RyanMcpherson141** [comment]
Facing same issue. Please let me know when it is fixed or if there is a temp fix for it.

**Comment by anubra266** [comment]
Fixed in v4.5.8 @RyanMcpherson141 @jamesopti @eric-hc
